Library/Formula/sslsplit.rb

```diff
@@ -0,0 +1,21 @@
+class Sslsplit < Formula
+  homepage "https://www.roe.ch/SSLsplit"
+  url "https://github.com/droe/sslsplit/archive/0.4.11.tar.gz"
+  sha256 "a33808f5d78000d1a919e61bbdfc48478ad40c1a9b09a0250b322f8954cc5ae0"
+
+  depends_on "check" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libevent"
+  depends_on "openssl"
+
+  def install
+    ENV.deparallelize
```

Comment:
Please add a comment pointing to https://github.com/droe/sslsplit/issues/81, thanks.